Speech_Recognition/src/Requirements.txt

```diff
@@ -0,0 +1,6 @@
+# Python version
+The "SpeechRecognition" library, which is commonly used for speech recognition in Python, is compatible with Python 3.x versions. It's recommended to use Python 3.3 or later when working with this library.
+
+# Speech recognition libraries
+pyaudio==0.2.11        # For audio input/output
+SpeechRecognition==3.8.1  # For speech recognition functionality
```

Comment:
Please adapt your file.
the naming should be with only lowercase letter `requirements.txt`
Make sure to only include the libraries and it's version, comments around what are they used for can be added in the code part when you import them if you feel like it